app/components/Vault/index.js

```diff
@@ -763,34 +759,48 @@ const Vault = (props) => {
     vaultAssets = truncateUsd(0);
   }
 
-  // FIXME: migration vaults
-  const retired = [
-    '0x7Ff566E1d69DEfF32a7b244aE7276b9f90e9D0f6', // crvSBTC v1
-    '0x5334e150B938dd2b6bd040D9c4a03Cff0cED3765', // crvRENBTC v1
-    '0xBacB69571323575C6a5A3b4F9EEde1DC7D31FBc1', // crvSAAVE v1
-    '0x7F83935EcFe4729c4Ea592Ab2bC1A32588409797', // crvOBTC v1
-    '0x123964EbE096A920dae00Fb795FFBfA0c9Ff4675', // crvPBTC v1
-    '0xA8B1Cb4ed612ee179BDeA16CCa6Ba596321AE52D', // crvBBTC v1
-    '0x07FB4756f67bD46B748b16119E802F1f880fb2CC', // crvTBTC v1
-    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE', // ETH v1
-    '0xe1237aA7f535b0CC33Fd973D66cBf830354D16c7', // WETH v1
-    '0xcB550A6D4C8e3517A939BC79d0c7093eb7cF56B5', // WBTC v2
+  // These are our retired v1 vaults, shutting down but not migrating.
+  const retiredv1 = [
+    '0x29E240CFD7946BA20895a7a02eDb25C210f9f324', // aLINK v1
+    '0x881b06da56BB5675c54E4Ed311c21E54C5025298', // LINK v1
   ];
 
-  if (address === '0xBA2E7Fed597fd0E3e70f5130BcDbbFE06bB94fe1') {
-    // FIXME: yfi vault
-    apyRecommended = 'N/A';
-    apyTooltip = 'Inactive with YIP-56: Buyback and Build';
-  } else if (retired.includes(address)) {
+  // These are our retired v2 vaults, shutting down but not migrating.
+  const retiredv2 = [
+    '0xe11ba472F74869176652C35D30dB89854b5ae84D', // HEGIC v2
+  ];
+
+  const hackedOrToBeAbsolutelyRemoved = [
+    '0xeb231ca94f718daf8ef8e7333b0d690fab9c59d8bc91bc20b5515c77eaff947e', // hacked cySUSD
+  ];
```

Comment:
We can remove this since it's not actually a vault, it's for iron bank, so not relevant to v2 website

Reply:
I removed cySUSD but left the array empty just in case we may need it one day, that ok?